app.js

```diff
@@ -0,0 +1,37 @@
+express = require('express');
+http = require('http');
+
+app = express();
+
+api = express.Router()
+api.all(/(.*)/, function(req, res) {
+    // TODO: Verify data?
+    var options = {
+        // TODO: Handle any verb (check req.method)
+        host: process.env.API_HOST,
+        port: process.env.API_PORT,
+        path: req.path
+    };
+
+    var request = http.request(options, function(response) {
+        // TODO: Reserialize data?
+        response.setEncoding('utf8');
+        response.on('data', function(chunk) {
+            res.send(chunk);
+        });
+    });
+
+    request.end();
+});
+
+app.use('/api', api);
+
+app.get('/', function(req, res) {
+    res.set('Content-Type', 'text/plain');
+    res.send('Hello, world!');
+});
+
+var server = app.listen(80, function() {
```

Comment:
Not sure whether this is relevant in a docker context. Ultimately we are probably going to have some sort of front-end server (nginx?) so I'm not saying it's difficult to do if we do want to. In that case, lets use 8000 so it can be run alongside the platform docker container which exposes 8080 on the same (virtual) machine.